source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/data.tsx

```diff
@@ -9,20 +9,13 @@ import { withCollectionDataActions } from '@data/containers/resources/notificati
 
 const mapNetworkToProps = (passedProps) => {
   return {
-    notifications: [
-      q => q
-        .findRecords(NOTIFICATION)
-        .sort('-time', '-isViewed'),
```

Comment:
how is sort handled now?

Reply:
derp.  need to add that back in... it was breaking the request because those aren't real things...